### PR TITLE
Add genotype references for HCV

### PR DESCRIPTION
**Pull request by CBeelen**
HCV region coordinates are defined relative to different references than the seed references. I have added these references and made sure that the whole genome alignment is happening relative to them, to prevent shifts between the region in the seed and the reference used for the region coordinates (as described in #794).

### CODE REVIEW

File: micall/utils/fetch_sequences.py [hunk 1]
```diff
@@ -281,22 +281,27 @@ def check_hcv_seeds(project_config, unchecked_ref_names: set):
 downloaded from https://hcv.lanl.gov/content/sequence/NEWALIGN/align.html
 This script contains a complete list of the reference accession numbers.
 """)
-    source_ids = {'HCV-1a': 'Ref.1a.US.77.H77.NC_004102',
+    source_ids = {# 'HCV1A': 'NC_004102', SAME, all HCV1 seeds except B use this
+                  'HCV-1a': 'Ref.1a.US.77.H77.NC_004102',
+                   # 'HCV1B': 'AJ238799', DIFFERENT, used only for B
```
Comment (donkirkby) [inline micall/utils/fetch_sequences.py:286]: How about adding a separate method to check HCV genotype refs?

**Review by donkirkby — CHANGES_REQUESTED** [review]
I don't think you need the seed groups in the genotype references section.

File: micall/utils/fetch_sequences.py (hunk 1 above)
Comment (CBeelen) [reply micall/utils/fetch_sequences.py]: I've added a method to check our project genotype sequences against the online references - I think that's what you had in mind, right?

Comment (donkirkby) [reply micall/utils/fetch_sequences.py]: That is what I had in mind, but now I see that the same sequences are used in `check_hcv_coordinates()`. If it's easy to combine those two methods, please do. If it's a pain for some reason, just print a message at the start of the new method to say that it's using the same sequences as `check_hcv_coordinates()`. One of the main uses for this script is for me to come back to, some time in the future, and remember where we got our reference sequences from and why we chose them.

Comment (donkirkby) [reply micall/utils/fetch_sequences.py]: The new version looks much simpler. Merging.

**Pull request merged**